### PR TITLE
add section our team and advisor

### PR DESCRIPTION
**Pull request by marknguyen85**

### CODE REVIEW

File: src/assets/scss/components/aboutus.scss [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+  &__header {
+    padding: 60px 0 0;
+
+    &-who-we-are {
+      height: 29px;
+      font-family: Montserrat;
+      font-size: 24px;
+      font-weight: 500;
+      font-style: normal;
```
Comment (artwooood) [inline src/assets/scss/components/aboutus.scss]: Dont just copy & paste some css from zeplin. A lot of them are redundant. Please check again about the default style element in css

File: src/assets/scss/components/aboutus.scss [hunk 2]
```diff
@@ -0,0 +1,114 @@
+.aboutus {
```
Comment (artwooood) [inline src/assets/scss/components/aboutus.scss]: Can you make it to about-us. So do the file name

File: src/assets/scss/components/aboutus.scss [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+      font-style: normal;
+      font-stretch: normal;
+      line-height: normal;
+      letter-spacing: normal;
+      text-align: center;
+      color: #7a82a9;
+    }
+  }
+
+  @media (max-width: 800px) {
```
Comment (artwooood) [inline src/assets/scss/components/aboutus.scss]: We should only use breakpoint as variable for consistancy

File: src/assets/scss/components/advisor.scss [hunk 4]
```diff
@@ -0,0 +1,121 @@
+.advisor {
```
Comment (artwooood) [inline src/assets/scss/components/advisor.scss]: Is there a different between team member & advisory? Can we make it a common component with the same css style?

File: src/components/layouts/AboutUs.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+                <div className={"container"}>
+                    <div className={"aboutus__header"}>
+                        <div className={"aboutus__header-who-we-are"}>Our Team</div>
+                        <div className={"aboutus__header-description"}>Lorem ipsum dolor sit amet, consectetur adipiscing elit. In facilisis sollicitudin ultricies. Nam viverra urna quis vulputate pulvinar.</div>
+                    </div>
+                    <div className={"aboutus__container"}>
+                        <div className={"aboutus__team-members"}>
+                            <div className="aboutus__team-member aboutus__team-team">
+                                <div className="aboutus__team-member-avatar aboutus__team-member-avatar--bio" 
+                                    data-bio="..."
```
Comment (artwooood) [inline src/components/layouts/AboutUs.js]: Please remove all data attribute.

File: src/components/layouts/AboutUs.js [hunk 6]
```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react'
+
+class AboutUs extends Component {
```
Comment (artwooood) [inline src/components/layouts/AboutUs.js:3]: Can we separate team members and advisors into two components?

File: src/components/layouts/AboutUs.js [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+                                    <div className="aboutus__team-member-name">Loi Luu</div>
+                                    <div className="aboutus__team-member-title">CEO & Co-Founder of Kyber Network</div>
+                                </div>
+                                <div className="aboutus__team-member aboutus__team-team">
+                                    <div className="aboutus__team-member-avatar aboutus__team-member-avatar--bio" 
+                                        data-bio="..." 
+                                        data-modal-id="bio-modal">
+                                        <img src={require("../../assets/images/aboutus/victor_tran.jpg")}></img>
+                                    </div>
+                                    <div className="aboutus__team-member-name">Victor Tran</div>
```
Comment (manhlx3006) [inline src/components/layouts/AboutUs.js]: Should change to `Head of Development & Co-Founder of Kyber Network`

**Comment by manhlx3006** [comment]
Please also change `Why Us` to `About Us`

File: src/assets/scss/components/aboutus.scss (hunk 3 above)
Comment (marknguyen85) [reply src/assets/scss/components/aboutus.scss]: dont understand

File: src/assets/scss/components/aboutus.scss (hunk 1 above)
Comment (marknguyen85) [reply src/assets/scss/components/aboutus.scss]: right but i am not follow project from start so it is difficult for me can find class what were define in the past.

Comment (artwooood) [reply src/assets/scss/components/aboutus.scss]: I mean some styles likes font-stretch, font-style, line-height, letter-spacing, etc is normal as default. You can remove them all

File: src/assets/scss/components/aboutus.scss (hunk 3 above)
Comment (artwooood) [reply src/assets/scss/components/aboutus.scss]: We have some breakpoint variables in variables.scss file

File: src/components/layouts/AboutUs.js (hunk 6 above)
Comment (artwooood) [reply src/components/layouts/AboutUs.js:3]: I dont think AboutUs can be placed in layout folder... Can you move it to another folder?

File: src/assets/scss/components/advisor.scss (hunk 4 above)
Comment (artwooood) [reply src/assets/scss/components/advisor.scss]: Can you write this to about-us.scss and dele this file

File: src/assets/scss/components/aboutus.scss (hunk 1 above)
Comment (marknguyen85) [reply src/assets/scss/components/aboutus.scss]: ah, ok thanks for your suggestion

File: src/assets/scss/components/aboutus.scss [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+  }
+  
+  &__team-members {
+    width: 80%;
+    margin: 30px auto 10px auto;
+    text-align: center;
+  }
+
+  &__team-member {
+    width: 23%;
```
Comment (artwooood) [inline src/assets/scss/components/aboutus.scss]: we should make it 22% of width for each member & margin-right: 4%. Except the 4 last member, margin-right should be 0

File: src/assets/scss/components/aboutus.scss [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+      font-family: Montserrat;
+      font-size: 16px;
+      font-weight: 600;
+      text-align: center;
+      color: #aab1d4;
+      margin: 15px 0;
+    }
+
+    &-title {
+      height: 18px;
```
Comment (artwooood) [inline src/assets/scss/components/aboutus.scss]: please remove this style to make the height auto

File: src/components/layouts/Header.js [hunk 10]
```diff
@@ -28,7 +28,7 @@ class Header extends Component {
           </div>
           <div className={"header__content"}>
             <a href="/" className={"header__content-item active"}>Swap</a>
-            <a href="/" className={"header__content-item"}>About Us</a>
+            <a href="#aboutus" className={"header__content-item"}>About Us</a>
```
Comment (manhlx3006) [inline src/components/layouts/Header.js:31]: Should also add to `Footer.js`

File: src/components/layouts/AboutUs.js [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+                            <div className="about-us__team-member aboutus__team-team">
+                                <div className="about-us__team-member-avatar aboutus__team-member-avatar--bio">
+                                    <img alt={"Mike Le"} src={require("../../assets/images/aboutus/mike_le.png")}></img>
+                                </div>
+                                <div className="about-us__team-member-name">Mike Le</div>
+                                <div className="about-us__team-member-title">CEO & Co-Founder</div>
+                            </div>
+                            <div className="about-us__team-member aboutus__team-team">
+                                <div className="about-us__team-member-avatar aboutus__team-member-avatar--bio">
+                                    <img alt={"Mark Nguyen"} src={require("../../assets/images/aboutus/mark_nguyen.jpg")}></img>
```
Comment (manhlx3006) [inline src/components/layouts/AboutUs.js:23]: can also update your photo, photos of Long Vuong and Tu Nguyen to png

File: src/components/layouts/Header.js (hunk 10 above)
Comment (marknguyen85) [reply src/components/layouts/Header.js:31]: ok i forgot

**Review by artwooood — APPROVED** [review]

File: src/assets/scss/components/aboutus.scss [hunk 12]
```diff
@@ -0,0 +1,106 @@
...
+
+  @media (max-width: $medium-breakpoint) {
+    &__team-member {
+      width: 28%;
+    }
+  }
+
+  @media (max-width: $small-breakpoint) {
+    &__team-member {
+      width: 45%;
```
Comment (artwooood) [inline src/assets/scss/components/aboutus.scss:99]: We have still have margin-right: 0 at `4n+1` for this element. Please make to `2n+1`

**Pull request merged**